### DISCUSSION TITLE
Speed up time based events

### DISCUSSION THREAD

**Discussion by Damaj301damaj-lol** [rafaelvcaetano/melonDS-android#765] [Ideas]
It would be nice if there was a setting that allowed you to speed up time based events while fast forwading :D

**Comment by rafaelvcaetano** [comment]
That will only be possible once the PC version adds support for it. They have been working on it. You can track the progress here: https://github.com/Arisotura/melonDS/pull/1237

**Reply by Damaj301damaj-lol** [reply]
Oh okay, got it!
